.ci/openjdk14-excluded.files

```diff
@@ -58,7 +58,7 @@
 
 <!-- until https://github.com/checkstyle/checkstyle/issues/8653 -->
 <module name="BeforeExecutionExclusionFileFilter">
-  <property name="fileNamePattern" value="[\\/]TrailingComma.java$"/>
+  <property name="fileNamePattern" value="enum[\\/]TrailingComma.java$"/>
```

Comment:
https://github.com/checkstyle/checkstyle/pull/9229 has similar fix for this line since there are 2 files with same name that cause 2 different failures. After merge of this or other PR, it should be removed completely